modules/service/src/test/scala/lucuma/odb/graphql/mutation/createObservation.scala

```diff
@@ -27,6 +32,19 @@ import java.time.LocalDateTime
 
 class createObservation extends OdbSuite with CreateProgramOps with LinkUserOps with SetAllocationOps with CreateObservationOps {
 
+  extension (ac: ACursor)
+    def downPath(p: String*): ACursor =
+      p.toList match {
+        case Nil    => ac
+        case h :: t => ac.downField(h).downPath(t*)
+      }
+
+    def liftIO[A: Decoder]: IO[A] =
+      ac.as[A].leftMap(f => new RuntimeException(f.message)).liftTo[IO]
+
+    def downIO[A: Decoder](p: String*): IO[A] =
+      downPath(p*).liftIO[A]
```

Comment:
Something like this might be generally useful in the test cases?

Reply:
Yeah maybe we could move this up into `OdbSuite` or something.